### PR TITLE
refactor adapter deployer

### PR DESCRIPTION
**Pull request by ilinzweilin**

### CODE REVIEW

File: src/lender/adapters/deployer.sol [hunk 1]
```diff
@@ -60,59 +60,33 @@ contract AdapterDeployer {
...
-
-    function deploy(bool wireMgr) public {
-		setupMkrAdapter(wireMgr);
-    }
-
-    function deploy() public {
-        deploy(false);
-    }
+    function deployClerk() public {
+        require(lenderDeployer.seniorToken() != address(0));
```
Comment (offerijns) [inline src/lender/adapters/deployer.sol:72]:
```suggestion
        require(lenderDeployer.clerk() != address(0));
```

**Review by offerijns — CHANGES_REQUESTED** [review]

File: src/lender/adapters/deployer.sol (hunk 1 above)
Comment (ilinzweilin) [reply src/lender/adapters/deployer.sol]: I guess you mean `require(lenderDeployer.clerk() == address(0));`

Comment (ilinzweilin) [reply src/lender/adapters/deployer.sol]: seniorToken check can stay, as it makes sure that the contract has to be deployed prior to deploying the clerk

**Review by offerijns — APPROVED** [review]
LGTM

**Pull request merged**